instrumentation/opentelemetry-instrumentation-django/tests/test_middleware.py

```diff
@@ -343,6 +532,16 @@ def test_span_name_404(self):
         span = span_list[0]
         self.assertEqual(span.name, "GET")
 
+    def test_nonstandard_http_method_span_name(self):
+        Client().request(
+            REQUEST_METHOD="NONSTANDARD", PATH_INFO="/span_name/1234/"
+        )
+        span_list = self.memory_exporter.get_finished_spans()
+        self.assertEqual(len(span_list), 1)
+
+        span = span_list[0]
+        self.assertEqual(span.name, "HTTP")
```

Comment:
I think here would be nice to also assert the http method is `_OTHER`.
Also, as suggestion would be nice to add a test case to check in new semconv where nonstandard http also requires the `http.request.method_original`